client/components/data-labels/labels.directive.js

```diff
@@ -90,6 +101,57 @@ angular.module('probedock.dataLabels').directive('simpleLabel', function() {
       return str;
     }
   };
+})
+.controller('SourceUrlLabelCtrl', function($scope) {
+  _.defaults($scope, {
+    tooltipPlacement: 'top'
+  });
+
+  $scope.isDirty = function() {
```

Comment:
May we should rename this to `isNotPushed` ou `isNotSynced` to avoid the confusion with `$scope.scm.dirty`. What do you think?